src/web-components/Button/index.ts

```diff
@@ -17,56 +20,58 @@ export class SharedButtonBase extends LitElement {
     buttonClass: unknown;
 
     render() {
+        console.log(this.buttonColor);
+        console.log(this.textColor);
         return html`
-            <button class="shared-button ${this.buttonClass}">
+            <button class="lit-button ${this.buttonClass}" style="background-color: ${this.buttonColor}; color:${this.textColor};">
                 <slot></slot>
             </button>
         `;
     }
 }
 
-export class SharedButtonSmall extends SharedButtonBase {
-    static renderTag = () => html`<shared-button-small></shared-button-small>`;
+export class LitButtonSmall extends LitButtonBase {
```

Comment:
Let's keep the name "SharedButton" instead of "LitButton". This is because we may use this library to render react elements